docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
     command: [ "./deploy/wait-for-postgres.sh", "bash", "/label-studio/deploy/start_label_studio.sh" ]
 
   db:
-    image: postgres:11.5
+    image: postgres:10.16
```

Comment:
why?

Reply:
Previously I had a problems with pg_hba.conf patch, but it works now with 11.5.